src/server/scripts/Northrend/zone_zuldrak.cpp

```diff
@@ -1224,4 +1328,5 @@ void AddSC_zuldrak()
     RegisterSpellScript(spell_zuldrak_zuldrak_rat);
     RegisterSpellScript(spell_zuldrak_gymers_grab);
     RegisterSpellScript(spell_zuldrak_gymers_throw);
+    new npc_servant_of_drakuru();
```

Comment:
Please use new register model with RegisterCreatureAI, you can find examples in this file